cmd/api/src/analysis/ad/ntlm_integration_test.go

```diff
@@ -155,74 +155,136 @@ func TestPostNTLMRelaySMB(t *testing.T) {
 	// TODO: Add some negative tests here
 	testContext := integration.NewGraphTestContext(t, graphschema.DefaultGraphSchema())
 
-	testContext.DatabaseTestWithSetup(func(harness *integration.HarnessDetails) error {
-		harness.NTLMCoerceAndRelayNTLMToSMB.Setup(testContext)
-		return nil
-	}, func(harness integration.HarnessDetails, db graph.Database) {
-		operation := analysis.NewPostRelationshipOperation(context.Background(), db, "NTLM Post Process Test - CoerceAndRelayNTLMToSMB")
+	t.Run("NTLMCoerceAndRelayNTLMToSMB Success", func(t *testing.T) {
```

Comment:
This change just wrapped existing integration test with a `t.Run` in order to add the new self relay test in a clean and self contained scope